{"description": "An issue was discovered in OpenCV 4.1.0 (OpenCV-Python 4.1.0.25). There is a divide-by-zero error in cv::HOGDescriptor::getDescriptorSize in modules/objdetect/src/hog.cpp.", "methods": ["Attempted to cause a divide-by-zero error in the OpenCV library."]}